c7n/resources/rds.py

```diff
@@ -1568,3 +1568,44 @@ def process(self, resources, event=None):
                     results.append(resource)
                     break
         return results
+
+
+@actions.register('modify-protect')
```

Comment:
we should just have a modify-db action that allows for setting this or other db configurations, not separate actions per possible parameter permutation to that single api.

Reply:
added